packages/xo-server-audit/src/index.js

```diff
@@ -245,6 +277,41 @@ class AuditXoPlugin {
       }))
   }
 
+  // See www-xo#344
+  async _automaticIntegrityCheck() {
```

Comment:
This feature is not an automatic check, maybe `uploadLasthash`?

Reply:
But this methods has two roles, checking the integrity of stored records then update the last hash.

Reply:
`IntegrityCheckWithLastHashUpload`?